src/dsvision.py

```diff
@@ -0,0 +1,97 @@
+import cv2
+
+from constants.CameraConstants import kRearCamera
+
+
+def Initialization():
+  TargetSize = 13  # Note (foam ring) diameter in inches
+
+  global guidelineSpacing
+  guidelineSpacing = int(kRearCamera.FL * TargetSize / kRearCamera.MinRange)
+  global guidelineXOffset
+  guidelineXOffset = int(kRearCamera.FL * (kRearCamera.y / kRearCamera.MinRange))
+
+
+def get_capture(window_name, video_capture_device_index=0):
+  # localsim = "http://localhost:1181/?action=stream"
+  roborio = "http://roboRIO-3985-frc.local:1181/?action=stream"  # IP depends on connection method (USB Tether, Ethernet, Radio)
```

Comment:
If the limelight is ever added, this url might change.

A part of the limelight setup is to set the rio to a static ip. This new static ip would adhere to the [team ip notation](https://docs.wpilib.org/en/stable/docs/networking/networking-introduction/ip-configurations.html#te-am-ip-notation).

Reply:
This is for the rear camera. The limelight will be mounted forwards.